cmd/testwrapper/flakytest/flakytest.go

```diff
@@ -41,4 +50,24 @@ func Mark(t testing.TB, issue string) {
 		fmt.Fprintf(os.Stderr, "%s: %s\n", FlakyTestLogMessage, issue)
 	}
 	t.Logf("flakytest: issue tracking this flaky test: %s", issue)
+
+	// Record the root test name as flakey.
+	rootFlakesMu.Lock()
+	defer rootFlakesMu.Unlock()
+	mak.Set(&rootFlakes, t.Name(), true)
+}
+
+// Marked reports whether the current test or one of its parent was marked flaky.
```

Comment:
parents plural to match "one of"? 

Or is this some en_UK difference likes "Google are announcing earnings" instead of "Google is announcing earnings" or how zero is singular in French? :)

Reply:
I'm sorry to say it's just a typo.